File: regress/rust/json-null.rs
use orb::ort;
use std::env;

fn main() {
    let args: Vec<String> = env::args().collect();
    assert_eq!(args.len(), 2);
    let db = ort::Ortdb::new(&args[1]);
    let ctx = db.connect().unwrap();

    let id1 = ctx.db_foo_insert(None, None, None, None).unwrap();

    let obj1 = ctx.db_foo_get_id(id1).unwrap().unwrap();

    assert_eq!(obj1.export(), r#"{"a":null,"b":null,"c":null,"d":null,"id":"1"}"#);


}

File: regress/nodejs/json-null.ts
const db: ortdb = ort(dbfile);
const ctx: ortctx = db.connect();

const id1: bigint = ctx.db_foo_insert(null, null, null, null);
if (id1 < 0)
	return false;

const obj1: ortns.foo|null = ctx.db_foo_get_id(id1);
if (obj1 === null)
	return false;

if (JSON.stringify(obj1.export()) !== '{"a":null,"b":null,"c":null,"d":null,"id":"1"}')
	return false;

return true;
